orchagent/aclorch.cpp

```diff
@@ -1615,6 +1690,10 @@ shared_ptr<AclRule> AclRule::makeShared(AclOrch *acl, MirrorOrch *mirror, DTelOr
         {
             return make_shared<AclRulePacket>(acl, rule, table);
         }
+        else if (acl->isUsingEgrSetDscp(table) || table == EGR_SET_DSCP_TABLE_ID)
+        {
+            return make_shared<AclRuleUnderlaySetDhcp>(acl, rule, table, m_metadataMgr);
+        }
```

Comment:
`AclRuleUnderlaySetDhcp` should be a typo?

Reply:
fixed